src/main/java/com/greghaskins/spectrum/ConstructorBlock.java

```diff
@@ -16,14 +18,19 @@ public void run() throws Throwable {
     try {
       final Constructor<?> constructor = this.klass.getDeclaredConstructor();
       constructor.setAccessible(true);
-      constructor.newInstance();
+      testObject = constructor.newInstance();
```

Comment:
Yeah - I made that. It would be terrible in a parallel running scenario... potentially!